Src/lib/UL/uart_manager/uart_manager.c

```diff
@@ -34,22 +37,22 @@ e_uart_manager_err_t uart_manager_deinit()
 
 e_uart_manager_err_t convert_header(led_num_t led_num)
 {
-    package.header.led_num = (uint8_t)led_num;
+    send_package.send_header.led_num = (uint8_t)led_num;
```

Comment:
If you defined struct as send_package seems you don't need to call field as send_header, just rename it to "header"